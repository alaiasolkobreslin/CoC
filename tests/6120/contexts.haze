
let ctx1 = forall X:Type, forall X:Type, forall z:(forall _:X, Type), Type in
let ctx2 = forall X:Type, forall x:X, forall y:(forall _:X, forall _:X, Type), Type in

ctx1