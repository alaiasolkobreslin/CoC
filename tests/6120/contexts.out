contexts.haze

Let: ctx1 beta reduced: (forall X:Type , (forall X:Type , (forall z:(forall _:X , Type) , Type)))

Let: ctx2 beta reduced: (forall X:Type , (forall x:X , (forall y:(forall _:X , (forall _:X , Type)) , Type)))
(forall X:Type , (forall X:Type , (forall z:(forall _:X , Type) , Type)))
Term typed: Type

